sonar-project.properties

```diff
@@ -1,4 +1,3 @@
-sonar.projectKey=populationgenomics_metamist_AZHVO0XpdaNSak8FrtKn
-sonar.python.version=3.11
```

Comment:
Not sure we want to remove the python version?

Reply:
Sorry seems like I selected more than I intended to remove, good spot, fixed now!